src/main/java/com/mars/specialty/application/SpecialtyService.java

```diff
@@ -0,0 +1,32 @@
+package com.mars.specialty.application;
+
+import com.mars.specialty.domain.Specialty;
+import com.mars.specialty.domain.SpecialtyNotFoundException;
+import com.mars.specialty.infrastructure.SpecialtyJpaRepository;
+
+public class SpecialtyService {
```

Comment:
앗 빼먹었는데 Spring bean으로 등록하기 위해 @Service를 붙여줘야합니다~
```suggestion
@Service
public class SpecialtyService {
```